docs/rules/jsx-first-prop-new-line.md

```diff
@@ -6,11 +6,12 @@ Ensure correct position of the first property.
 
 ## Rule Details
 
-This rule checks whether the first property of all JSX elements is correctly placed. There are three possible configurations:
+This rule checks whether the first property of all JSX elements is correctly placed. There are four possible configurations:
```

Comment:
maybe instead of enumerating "four", we should just say "These are the possible configurations:" and then nobody needs to update it in the future?